src/screens/OrgAdminHomePage/OrgAdminHomePage.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import OrgAdminNavbar from 'components/OrgAdminNavbar/OrgAdminNavbar';
+import styles from './OrgAdminHomePage.module.css';
+import web from 'assets/talawa-logo-lite-200x200.png';
+import web1 from 'assets/174857.png';
+import web2 from 'assets/Dribbble-icon-Logo-PNG-Image.png';
```

Comment:
Please use descriptive names for your asset imports.

Since there are multiple imports in this scenario you will have to name them accordingly.

Such as
```suggestion
import TalawaLogo from 'assets/talawa-logo-lite-200x200.png';
import LinkedinLogo from 'assets/174857.png';
import DribbleLogo from 'assets/Dribbble-icon-Logo-PNG-Image.png';
```